### ISSUE TITLE
Invalid constant name "application" (invalid-name)

### ISSUE THREAD

**Issue by craiga** [PyCQA/pylint-django#77]
`pylint_django` complains about Django-generated code:

```
$ django-admin startproject testproj
$ cd testproj
$ pylint --load-plugins pylint_django testproj
No config file found, using default configuration
************* Module testproj.wsgi
C: 16, 0: Invalid constant name "application" (invalid-name)
…
```

Using: pylint==1.5.6 and pylint-django==0.7.2.

**Comment by jasjuang** [comment]
Ran into the same problem. Would be great to see this fixed.

**Comment by sieira** [comment]
+1

**Comment by cactail** [comment]
Same problem here. Should I ignore the wsgi.py module?

**Comment by bittner** [comment]
@cactail 

As an alternative to disabling the message or ignoring the whole module, using PyLint's `--good-names` may be a soft enough option:

```bash
$ pylint testproj --load-plugins=pylint_django --good-names=application
```
As usual, those options can also go into a [PyLint configuration file](https://pylint.readthedocs.io/en/latest/user_guide/run.html#command-line-options) to save typing on the command line:
```ini
# FILE: pylintrc  or .pylintrc
load-plugins = pylint_django
good-names = application
```

@carlio 

Where would code to ignore `"application" (invalid-name)` have to go? `augmentations` [looks like the right place](https://github.com/landscapeio/pylint-django/blob/master/pylint_django/augmentations/__init__.py#L634), but it's really not all that obvious what to do. Can you give a hint?

**Comment by bittner** [comment]
Any hint to start tackling this issue would be greatly appreciated, @carlio!

Who will take care of maintenance and PRs in future, now that this repo is under the PyCQA umbrella?

**Comment by atodorov** [comment]
@bittner @colinhowe and myself joined this project as new maintainers just before it moved under PyCQA. I have closed a few issues yesterday and will continue to work on the rest during the next few weeks. 

Not sure what my permissions on the repo are after the move but I will gladly continue maintaining this package.

**Comment by carlio** [comment]
@bittner @atodorov @colinhowe   I'll will certainly be helping and now others are too and hopefully with the new moves it will help keep things going :-) It was a bit staid when I was on it alone so I think this is a good move for users and the lib itself.

**Comment by bittner** [comment]
Okay, thanks!

Remains the first part of the question to be answered:  :smirk: 

> Any hint to start tackling this issue would be greatly appreciated, @carlio!

**Comment by atodorov** [comment]
@bittner the easiest way will be to append 'application' to good_names in `pylint_django/plugin.py`. 

The other will be to add an augmentation which silences the invalid-name message from pylint when visiting a AssignName node, which is named "application" and lives inside the "wsgi" module. ATM I don't know if astroid nodes carry module information with them but quick look tells me no.
